tests/test_aaa_authorization_service.rb

```diff
@@ -22,16 +22,17 @@ class TestAaaAuthorizationService < CiscoTestCase
 
   def setup
     super
-    cleanup_aaa if @@pre_clean_needed
-    @@pre_clean_needed = false # rubocop:disable Style/ClassVars
-    feature_tacacs
+    if @@pre_clean_needed
```

Comment:
After looking at it more, it doesn't seem that the guard clause applies here.  The `setup` method would skip the actual setup if `@@pre_clean_needed` is set to `true`